core/templates/pages/exploration-editor-page/editor-tab/state-name-editor/state-name-editor.component.ts

```diff
@@ -54,6 +54,7 @@ angular.module('oppia').component('stateNameEditor', {
         var stateName = StateEditorService.getActiveStateName();
         StateNameService.setStateNameEditorVisibility(true);
         StateNameService.setStateNameSavedMemento(stateName);
+        ctrl.maxLen = 50;
```

Comment:
Hm, can we move this to a constant in `constants.ts` and then use that in `state-name-editor.component.html` as well as `outcome-destination-editor.directive.html`? This way, if we ever want to change the limit, we just have to change it in one place and it will reflect everywhere it is used.

Reply:
Done!